tests/test-verify-volume-size.yml

```diff
@@ -48,6 +48,6 @@
     var: storage_test_expected_size
 
 - assert:
-    that: storage_test_actual_size.bytes == storage_test_expected_size|int
+    that: (storage_test_expected_size|int - storage_test_actual_size.bytes) / storage_test_expected_size|int < 0.1
```

Comment:
1 percent is `0.01` - also suggest using `abs` in case `storage_test_actual_size.bytes` is larger than `storage_test_expected_size`
```
     that: (storage_test_expected_size|int - storage_test_actual_size.bytes)|abs / storage_test_expected_size|int < 0.01
```

Reply:
haha, thanks. That's a dopey mistake. I was planning to add the `abs` as well, but got too eager to see if it fixes the bug. Will do.